app/lib/frontend/handlers/headers.dart

```diff
@@ -57,8 +59,26 @@ class CacheHeaders {
     return headers.containsKey(HttpHeaders.cacheControlHeader);
   }
 
-  /// Default private-only caching.
-  static final defaultPrivate = CacheHeaders._(Duration.zero, private: true);
+  /// Private-only max-age zero.
+  static final privateZero = CacheHeaders._(Duration.zero, private: true);
+
+  /// Default cache-control for public user-interface
+  static final defaultPublicUI = CacheHeaders._(
+    Duration(minutes: 10),
+    public: true,
+  );
+
+  /// Default cache-control for user-interface
+  static final defaultUI = CacheHeaders._(
+    Duration(minutes: 10),
+    // public / private is inferred from authentication + session header
+  );
+
+  /// Default cache-control for API responses
+  static final defaultApi = CacheHeaders._(
+    Duration(minutes: 30),
+    // public / private is inferred from authentication + session header
```

Comment:
Isn't all of /api public?

Reply:
No, many endpoints are only for limited scope, e.g. admins (like, listing publisher members).

Reply:
Ah - of course. Nvm